package.json

```diff
@@ -29,7 +29,11 @@
     "test-cli": "NODE_ENV=production NODE_PATH=. _mocha --require ./tests/spec/specSetup.js --opts ./tests/config/mocha.opts 'bin/**/*.spec.js'",
     "test-unit": "NODE_ENV=production NODE_PATH=. _mocha --require ./tests/spec/specSetup.js --opts ./tests/config/mocha.opts '{src,tests/e2e,components}/**/*.spec.js'",
     "test-e2e": "TOGA_SERVER_PORT=3001 npm run nightwatch -- --components=./tests/e2e",
-    "nightwatch": "nightwatch -o ./tests/e2e/tests_output -c ./tests/config/nightwatch.conf.js"
+    "nightwatch": "nightwatch -o ./tests/e2e/tests_output -c ./tests/config/nightwatch.conf.js",
+    "preinstall": "npx npm-force-resolutions"
```

Comment:
is this because of an error like in here https://stackoverflow.com/questions/55921442/how-to-fix-referenceerror-primordials-is-not-defined-in-node?

Reply:
yes